qds_sdk/connection.py

```diff
@@ -43,20 +43,23 @@ def __init__(self, auth, base_url, skip_ssl_cert_check, reuse=True):
             self.session = requests.Session()
             self.session.mount('https://', MyAdapter())
 
-    @retry((RetryWithDelay, requests.Timeout), tries=6, delay=30, backoff=2)
```

Comment:
@mindprince 
1. currently we are raising RetryWithDelay only for 449, 502, 503, 504 error codes. should we do it for all 5xx codes?
2.  what should be the value of tries?